include/psa/crypto_types.h

```diff
@@ -120,6 +120,7 @@ typedef uint32_t psa_key_lifetime_t;
  * psa_key_id_t in crypto_platform.h instead of here. */
 #if !defined(MBEDTLS_PSA_CRYPTO_KEY_FILE_ID_ENCODES_OWNER)
 typedef uint32_t psa_key_id_t;
+#define PSA_KEY_ID_INIT 0
```

Comment:
Why do we need this?

Reply:
I was hoping to be able to use it in the assignment later on, without having to add an ifdef for MBEDTLS_PSA_CRYPTO_KEY_FILE_ID_ENCODES_OWNER in psa_set_key_lifetime. That didn't pan out.

Reply:
Is there any reason why we shouldn't remove it?

Reply:
I did end up using it in the initialization macros for key attributes.

Reply:
But we have it in `crypto_platform.h` already, is there any reason why we would want to maintain both definitions?

Reply:
I've removed the extra one from `crypto_platform.h`. Now, we have the INIT macro next to the typedef in all cases.